client/lib/plans/features-list.js

```diff
@@ -298,7 +298,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_BLOG_DOMAIN ]: {
 		getSlug: () => constants.FEATURE_BLOG_DOMAIN,
 		getTitle: () =>
-			i18n.translate( 'Free .blog Domain for One Year', {
+			i18n.translate( 'Free .blog Domain for one year', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.